index.html

```diff
@@ -41,30 +42,66 @@
           </li>
         </ul>
         <!--* hamburger -->
-        <div
-          class="w-[26px] h-[30px] border-white flex flex-col justify-center items-center cursor-pointer md:hidden"
+        <ul
+          id="bars"
+          class="w-[26px] h-[30px] flex flex-col justify-center items-center cursor-pointer md:hidden bars"
```

Comment:
- [ ] Hi @martinkarugaba, when I click or tap the hamburger button, the mobile menu appears. This was very great :+1: But when I scroll down to the bottom of the page and then click this button, the mobile menu appears but it was hidden and I could only see the menus by scrolling to the top. Can you try this out? Let's kindly make sure that these menus appear when the hamburger button is clicked.